label/models/label_print.py

```diff
@@ -66,6 +65,9 @@ def onchange_model(self):
                     if model_ids:
                         model_list.append(key)
         self.model_list = model_list
+        # TODO: this avoid crashing the app but may not be correct
```

Comment:
I think this whole method `onchange_model` is not useful and can be removed (the field in the view is invisible). Try to remove the method and see if it still works.